libs/wingsdk/src/target-tf-aws/api.ts

```diff
@@ -35,6 +45,12 @@ export class Api extends cloud.Api {
     });
   }
 
+  public get url(): string {
+    const region = (App.of(this) as App).region;
+    const awsSuffix = "amazonaws.com"; // TODO: use dns_suffix from https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/partition
+    return `https://${this.api.api.id}.execute-api.${region}.${awsSuffix}/${STAGE_NAME}`;
```

Comment:
Are you telling me there's no attribute for the url of the api gateway?

Reply:
You know what -- I thought there wasn't because I was looking at the [ApiGatewayRestApi resource](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/api_gateway_rest_api) docs, but after giving a second look, it looks like the [ApiGatewayDeployment resource](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_deployment) has an attribute with the invoke url, so we should be able to use that instead. Good callout